src/components/sidebar/components/sidebar-nav/sidebar-nav-menu-item/index.tsx

```diff
@@ -84,8 +86,17 @@ const SidebarNavSubmenu: React.FC<{
 
 const SidebarNavMenuItem: React.FC<SidebarMenuItemProps> = ({ item }) => {
   // TODO: the designs don't currently show a divider
+  // TODO: Update 2022-01-03: product home page, eg /waypoint, does
+  // TODO: show divider. Including a basic implementation for now,
+  // TODO: this is something we likely need to revisit with design.
   if (item.divider) {
-    return null
+    return <hr className={s.divider} />
+  }
+
+  // TODO: 2022-01-03: designs show a heading on the product home page,
```

Comment:
Have added `TODO`s here to signal that these features were implemented for the demo, so not with the usual rigour.

Reply:
I think this'll stay long term, or a version of it will, because it's possible we'll be replacing dividers with text headers. It'll definitely help us if that's the case!